web/themes/custom/hoeringsportal/assets/css/module/_search.scss

```diff
@@ -66,36 +70,36 @@
     }
   }
 
-  .fa-fw {
-    position: absolute;
-    left: 0;
-    right: 0;
-    margin: 0 auto;
-    top: 50%;
-    bottom: 0;
-    transform: translateY(-50%);
-  }
-
   .fa-times {
     display: none;
+    position: absolute;
+    top: 15px;
+    right: 3px;
   }
 
   .fa-search {
     display: block;
+    position: absolute;
+    top: 11px;
+    right: 3px;
   }
 
   &.is-open {
+    border: 1px solid rgba(133, 133, 133, .2);
+    background: $white;
+    padding-left: 1em;
```

Comment:
Maybe consider using the $spacer variable for paddings and margins. It equals 1rem.

Reply:
Changed.